### DISCUSSION TITLE
Add new language in Pimcore

### DISCUSSION THREAD

**Discussion by amarjeet6694** [pimcore/pimcore#12177] [Q&A / Support] [answered]
Is it possible to add new languages that are not currently present by default in Pimcore?
We have a client requirement to add few new languages e.g English (Mexico) [en_MX], English (China) [en_CN]

**Comment by Maksonis** [comment, marked as answer]
You can extend Pimcore LocaleService:

![image](https://user-images.githubusercontent.com/17277953/169512221-56d943f8-29a2-417e-8057-2496219924e0.png)

![image](https://user-images.githubusercontent.com/17277953/169512260-7cd3643c-e6df-4f34-bade-7c24b20a77a7.png)

**Reply by amarjeet6694** [reply]
Thanks @Maksonis for the quick response.

**Reply by studio1gmbh-pimcore** [reply]
@amarjeet6694 how did you solve it? The solution shown here does not work without a decorator, right?
